### PR TITLE
Portfolio website-Desktop version 🚀 

### PR DESCRIPTION
**Pull request by evansnyamekye**
Desktop Portfolio website 🌐. 
- Used CSS to design the layout of the portfolio website ☑️. 
- Applied appropriate dimensions to images, font, icons, etc. 🔍. 
- Implemented active hover effort on the contact button when a mouse cursor is placed on the button 🖱️. 
- Implemented active color on the button when the button is pressed by the user 🆑. 
- Used different specific images for both mobile and desktop contact sections 🖼️. 
- Used both Grid and Flex to place the cards on different screens. 💻

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -464,14 +466,50 @@ p {
...
+    padding: 5rem 9rem;
   }
 
   /* Work */
   .work .card {
     flex-direction: row;
-    gap: 40px;
+    gap: 25px;
+    height: 351px;
   }
```
Comment (joel-kalema) [inline styles.css]: - [ ] You did a good job @evansnyamekye 🥇, however, as I can see you gave your cards a `box-shadow` 
https://github.com/evansnyamekye/Post-index/blob/f7a77daf01eb8622da8a5a069050623872b666bd/styles.css#L185

But, I'll suggest you remove that line to stick to the given design. However the `box-shadow` will be used when we `hover` the card when are in the desktop version, it supposes to be something like this: 

````css
@media only screen and (min-width: 768px) {
       .
       .
       .

       .work .card {
            flex-direction: row;
            cursor: pointer;
            gap: 25px;
            height: 351px;
       }

      .work .card:hover {
            box-shadow: 0 10px 25px 0 rgba(0, 0, 0, 0.103);
            border: #A7AEFF solid 1px;
      }

      .
      .
      .
}

````

Kindly, fix it 😃

File: index.html [hunk 2]
```diff
@@ -1,351 +1,292 @@
...
+  <!-- End of header container -->
+
+  <!-- start of Main -->
+  <main>
+    <!-- start of Banner -->
+    <section class="banner">
+      <div class="container">
+        <div class="banner-text">
+          <h1>I’m Nyamekye. Glad to see you!</h1>
+          <p>
```
Comment (joel-kalema) [inline index.html]: - [ ] Also, make sure to aline your text as we can see the design given 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-07-13 134121| |Screenshot 2023-07-13 134102| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @evansnyamekye 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

TN6


## To highlight :champagne: 

- Your design looks good :100: :heavy_check_mark: 



## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @evansnyamekye , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Design
✅ ... Linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**